conf/test_full.yml

```diff
@@ -0,0 +1,32 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I added this because in the Import PR (#62), I added a parameter which nextflow will not recognize in the config. I discussed this in the nf-core slack channel and was told that this is a known problem and parameters should preferably be provided with a yml file and the -params-file argument

Reply:
Okay, I just found [that conversation](https://nfcore.slack.com/archives/CE6SDBX2A/p1675419156933019) on Slack, and I don't think the respondents had the full picture of what you were trying to do.

If we did need to activate additional tests with different sets of params, we wouldn't need to specify additional params files on the command line here. We could just create an additional profile to be specified, analagous to `test` and `test_full`. I'm pretty sure that would work fine, since the existing profiles are doing fine at overriding the things from `nextflow.config`.

Reply:
Yes, for the other configs it was no problem, but for some reason the `matrix_id_col` param I had to add in #62 was constantly overridden when I ran it with the .config so I had to switch to .yml which worked.

But in either case the ci has to be changed, no? If there is an easier way to add different tests, I'd of course be happy to use that

Reply:
I just saw your above comment, got it :)